src/tools/kdb/gen/elektra/elektragen.hpp

```diff
@@ -9,7 +9,7 @@
 #ifndef ELEKTRA_ELEKTRAGEN_HPP
 #define ELEKTRA_ELEKTRAGEN_HPP
 
-#include "template.hpp"
+#include "../template.hpp"
```

Comment:
It is better to avoid this. Ideally make an include folder for kdb/gen and add this folder to the search path for header files.